docs/design/router.md

```diff
@@ -109,6 +109,19 @@ Including this strategy means all timeBoundary queries are always routed to the
 
 Queries with a priority set to less than minPriority are routed to the lowest priority Broker. Queries with priority set to greater than maxPriority are routed to the highest priority Broker. By default, minPriority is 0 and maxPriority is 1. Using these default values, if a query with priority 0 (the default query priority is 0) is sent, the query skips the priority selection logic.
 
+#### queryContext
+
+This strategy reads the parameter `brokerService` from the query context and routes the query accordingly. If no valid `brokerService` is specified in the query context, the field `fallbackBrokerService` is used if set to a valid non-null value.
```

Comment:
can we call `fallbackBrokerService` `defaultBrokerService` instead? Latter looks more common in druid codebase.

Reply:
Renamed to `defaultManualBrokerService`